tasks/base/SLES-12/install_docker.yml

```diff
@@ -13,6 +13,11 @@
   until: remove_packages is success
 
 - name: Install docker
-  command: "zypper install -y --force-resolution --replacefiles {{ docker_version_map[docker_version]['package'] }}"
+  command: "zypper install -y --force-resolution --replacefiles {{ docker_version_map[docker_version]['package'] }} acl"
   args:
     warn: false
+
+- name: disable nscd (otherwise container will ignore their /etc/hosts)
```

Comment:
I think it would be better to remove the package instead of only disabling the daemon. That will open less opportunities for problems.